packages/ListBox/src/components/Box/Box.js

```diff
@@ -2,21 +2,26 @@ import React from "react";
 import PropTypes from "prop-types";
 import * as sc from "./Box.styles";
 import useListBox from "../../useListBox";
-
-const propTypes = {
-  /** Body content of the box. */
-  children: PropTypes.node.isRequired,
-};
+import { PropsContext } from "../../store/PropsProvider";
 
 export default function Box(props) {
-  const { children, ...moreProps } = props;
-  const [state] = useListBox();
+  const [{ triggerWidth }] = useListBox();
+  const { isInline, isReadOnly, hasError } = React.useContext(PropsContext);
 
   return (
-    <sc.Box data-pka-anchor="list-box-box" isInline={state.isInline} triggerWidth={state.triggerWidth} {...moreProps}>
+    <sc.Box
+      data-pka-anchor="list-box-box"
+      hasError={hasError}
+      isInline={isInline}
+      isReadOnly={isReadOnly}
+      triggerWidth={triggerWidth}
+    >
       {props.children}
     </sc.Box>
   );
 }
 
-Box.propTypes = propTypes;
+Box.propTypes = {
+  /** Body content of the box. */
+  children: PropTypes.node.isRequired,
+};
```

Comment:
I've moved `propTypes` and `defaultProps` to the bottom in several files to adhere to our new convention.